neutron/assetlist.json

```diff
@@ -532,6 +532,30 @@
       ]
     },
     {
+      "description": "GODDARD",
+      "denom_units": [
+        {
+          "denom": "factory/neutron1yqj9vcc0y73xfxjzegaj4v8q0zefevnlpuh4rj/GODDARD",
+          "exponent": 0,
+          "aliases": [
+            "ugoddard"
+          ]
+        },
+        {
+          "denom": "goddard",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/neutron1yqj9vcc0y73xfxjzegaj4v8q0zefevnlpuh4rj/GODDARD",
+      "name": "GODDARD",
+      "display": "goddard",
+      "symbol": "GODDARD",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/goddard.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/goddard.svg"
```

Comment:
The schema is broken here. Please use closing brackets for the goddard asset so it doesn't disrupt the BITCOSMOS asset, and please move to the bottom of the list.